post-types/class-post-type-commands.php

```diff
@@ -61,13 +91,13 @@ function ee_add_subcommands( $content ) {
 			$parent_page_id = $post_id;
 		}
 
-		$args = array(
+		$args = [
 			'post_type'      => $this->post_type,
 			'post_parent'    => $parent_page_id,
-			'posts_per_page' => 200,
+			'posts_per_page' => 200, // phpcs:ignore
```

Comment:
Why is it ignored? Add comment with specific rule.

Reply:
This actually throughs warning.

![Screenshot 2019-06-12 at 12 08 04 PM](https://user-images.githubusercontent.com/26354653/59328738-c6711580-8d0a-11e9-8eac-fe4205f4c590.png)

While running PHPCS I am not getting any specific rule with a warning (Please check above screenshot). Any idea how do I get a specific rule to ignore?